### DISCUSSION TITLE
Google colab runtime problem

### DISCUSSION THREAD

**Discussion by Rahul-i7** [TheLastBen/fast-stable-diffusion#2204] [Q&A]
First I would like to thank TheLastBen for fixing the gradio error but my colab runtime still keeps disconnecting. It's actually a big issue for me because i use colab's storage and move my models through codes from my google drive. So, when the runtime gets disconnected I lose all my files and have to redownload them and move my models again and this costs me a lot of time. Can anyone help me?
Using free colab btw...

**Comment by Rasad0** [comment]
he wouldn't do anything I also requested. simply we would not be able to use colab like before.

**Comment by TheLastBen** [comment]
colab free doesn't support sd anymore

**Reply by THX1139b** [reply]
They have also done something to the GPU setup. Now they have A100, V100 and T4 as well as two runtime types to choose from. Whatever that means for practical use, I notice that with the default settings I now eat through 100 computing units in about a working day whereas it used to last for many days of full use.

**Reply by TheLastBen** [reply]
use the T4 for SD

**Reply by THX1139b** [reply]
OK, thanks!
